projects/plugins/jetpack/extensions/blocks/event-countdown/edit.js

```diff
@@ -0,0 +1,130 @@
+import { useBlockProps } from '@wordpress/block-editor';
+import { Button, DateTimePicker, Dropdown, Placeholder } from '@wordpress/components';
+import { useInstanceId } from '@wordpress/compose';
+// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+import { dateI18n, __experimentalGetSettings } from '@wordpress/date';
+import { __ } from '@wordpress/i18n';
+import clsx from 'clsx';
+import moment from 'moment';
```

Comment:
I'm surprised this works, given that we do not import `moment` in the Jetpack plugin today:
https://github.com/Automattic/jetpack/blob/a0f4db2f8ea03a1089635509f47958add21a26c6/projects/plugins/jetpack/package.json#L46
Maybe that's because Gutenberg calls in `moment`?

Maybe that's something we should update (maybe we could move away from `moment` and rely on the core utilities instead?